src/test/scala/io/github/evis/scalafix/maven/plugin/params/CompiledDirectoryParamSpec.scala

```diff
@@ -12,10 +12,8 @@ class CompiledDirectoryParamSpec extends ParamSpec {
 
   it should "add both classpaths if invoked twice on the same Scalafix arguments builder" in {
```

Comment:
Now the test name is misleading, since CompiledDirectoryParams isn't invoked twice. It's invoked once, but with two paths.